index.html

```diff
@@ -4,42 +4,57 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
-    <script src="script.js"></script>
+    <title>Calculator</title>
+    <script src="script.js" defer></script>
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
     <div class="mainWrapper">
       <div class="windowWrapper">
-        <div class="inputWrapper">
-          <input class="inputWindow" type="text" />
+        <div class="topWrapper">
+          <div class="inputNumber">0</div>
         </div>
-        <!-- <div class="operatorWrapper">
-          <div class="operatorTop">
-            <button class="placeholder">+</button>
-            <button class="placeholder">-</button>
-            <button class="placeholder">*</button>
-            <button class="placeholder">/</button>
-          </div>
-        </div> -->
+
         <div class="numpadWrapper">
           <div class="topNumpad">
-            <button class="placeholder">7</button>
-            <button class="placeholder">8</button>
-            <button class="placeholder">9</button>
-            <button class="placeholder">4</button>
-            <button class="placeholder">5</button>
-            <button class="placeholder">6</button>
-            <button class="placeholder">1</button>
-            <button class="placeholder">2</button>
-            <button class="placeholder">3</button>
+            <button class="calcButton" id="button-side" data-clear>CA</button>
+            <button class="calcButton" id="button-side" data-plusminus>
+              +/-
+            </button>
+            <button class="calcButton" id="button-side" data-delete>⌫</button>
+            <button class="calcButton division" id="button-side" data-operation='division'>
+              ÷
+            </button>
           </div>
           <div class="bottomNumpad">
-            <button class="placeholder">0</button>
-            <button class="placeholder">,</button>
+            <div class="numbers">
+              <button class="calcButton" data-number='9'>9</button>
```

Comment:
For extra credit, you could try to get rid of the duplication here and show the number from the dataset via css.

Reply:
Did you mean js?

Reply:
With `html` and `css` only.
[spoiler](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes)

Reply:
Oh nice! I've added that with the latest commit :)